docs/components_page/components/input/colorpicker.py

```diff
@@ -0,0 +1,22 @@
+import dash_html_components as html
+from dash.dependencies import Input, Output
+import dash_bootstrap_components as dbc
+
+colorpicker = dbc.FormGroup(
+    [
+        dbc.Label("Choose the text color:", id="label"),
```

Comment:
If the user chooses white then the label completely disappears, maybe we could do something like

```python
dbc.Label(["Select a ", html.Span("color", id="color")])
```

and target just the word "color" with the callback?

Reply:
Good idea!